js/src/Lines.ts

```diff
@@ -670,16 +671,16 @@ export class Lines extends Mark {
     }
 
     dot: any;
-    legend_el: d3.Selection<SVGGElement, any, any, any>;
-    legend_line: d3.Line<[number, number]>;
-    legend_path_data: [number, number][];
+    legendEl: d3.Selection<any, any, any, any>;
+    legendLine: d3.Line<[number, number]>;
+    legendPathData: [number[]];
```

Comment:
Your compilation issue seems to be related to the type you changed here, `[number, number][]` was more specific so you might want to keep it that way.